### DISCUSSION TITLE
怎么退个款这么难呀！提交申请20的天了，还没退款

### DISCUSSION THREAD

**Discussion by ineedyou2008** [getlantern/lantern-forum-cn#746] [General]

**Comment by ineedyou2008** [comment]
![mmexport1722498024630](https://github.com/user-attachments/assets/f2b5474b-400a-4214-a824-79775c37129a)
![mmexport1722497881714](https://github.com/user-attachments/assets/d22beb5b-1164-4063-9b71-bd63b540618e)

**Comment by ineedyou2008** [comment]
我支付时，使用的邮箱是275693730@qq.com

**Reply by MaxMKMax** [reply]
@ineedyou2008 我帮您问一下. 在帮您重新提交问题. 最快下周给您答复. 谢谢.

**Comment by ineedyou2008** [comment]
好的， 非常感谢。







&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月1日(星期四) 下午3:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)





 
@ineedyou2008 我帮您问一下. 在帮您重新提交问题. 最快下周给您答复. 谢谢.

**Comment by ineedyou2008** [comment]
管理员，半个月时间又过去了！也没有等到你的一周后的回复。



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月1日(周四) 下午3:57
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)




 
@ineedyou2008 我帮您问一下. 在帮您重新提交问题. 最快下周给您答复. 谢谢.

**Reply by MaxMKMax** [reply]
@ineedyou2008 
您好. 请您提供我支付单据在这里. 上次连接的支付单据他们找不到了. 我马上给您安排紧急处理.

**Comment by MaxMKMax** [comment]
@ineedyou2008 
您今天给我提供支付单据的话今天应该可以帮您处理退款了. 不然要等到下周二了.

**Comment by ineedyou2008** [comment]
我通过微信二维码支付了340元，以下是我的支付凭证。
***@***.***作为用户名。
管理员请帮我处理，非常谢谢！

















&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月16日(星期五) 晚上6:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)





 
@ineedyou2008
 您今天给我提供支付单据的话今天应该可以帮您处理退款了. 不然要等到下周二了.

**Comment by ineedyou2008** [comment]
小牛哥
***@***.***






---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月16日(周五) 晚上6:12
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)




 
@ineedyou2008
 您今天给我提供支付单据的话今天应该可以帮您处理退款了. 不然要等到下周二了.

**Reply by MaxMKMax** [reply]
@ineedyou2008 您好. 您的单据和账号邮箱都被Gihub屏蔽了. 您发到我邮箱吧. 我帮您申请.
mike@getlantern.org

**Comment by MaxMKMax** [comment]
@ineedyou2008 您好. 公司想要知道您退款的理由. 您告述我好吗? 然后才能退款给您.

**Comment by ineedyou2008** [comment]
管理员，这又过去了11天了，退款进度到哪一步了呀？










小牛哥
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月30日(周五) 上午6:27
收件人: ***@***.***&gt;;
主题: 回复： [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)


原因是之前充值后，邮箱一直没有办法收到那个码。前后搞了半个月还是搞不定。因为工作需要，急需去到外面的网路。就重新买了其他的VPN了。所以要退费。







&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月30日(星期五) 凌晨0:19
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)





 
@ineedyou2008 您好. 公司想要知道您退款的理由. 您告述我好吗? 然后才能退款给您.

**Comment by ineedyou2008** [comment]
管理员，我需要的提供的资料，不管是你们流程变了还是网络的问题，我已经配合提交了很多遍了。提交了之后隔了一段时间让我提供退款理由我也提供了，时间又过了18天，我没有收到什么退款进度！退个款怎么这么难呀！！！能告诉我一个准确的退款时间吗？我们的彼此的时间都很宝贵呀！












小牛哥
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年9月11日(周三) 晚上10:59
收件人: ***@***.***&gt;;
主题: 回复： [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)


管理员，这又过去了11天了，退款进度到哪一步了呀？










小牛哥
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月30日(周五) 上午6:27
收件人: ***@***.***&gt;;
主题: 回复： [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)


原因是之前充值后，邮箱一直没有办法收到那个码。前后搞了半个月还是搞不定。因为工作需要，急需去到外面的网路。就重新买了其他的VPN了。所以要退费。







&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月30日(星期五) 凌晨0:19
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 怎么退个款这么难呀！提交申请20的天了，还没退款 (Discussion #746)





 
@ineedyou2008 您好. 公司想要知道您退款的理由. 您告述我好吗? 然后才能退款给您.
